app/locustio/jira/http_actions.py

```diff
@@ -234,9 +244,11 @@ def edit_issue_open_editor():
         assignee = fetch_by_re(params.issue_assigneee_reporter_pattern, content, group_no=2)
         reporter = fetch_by_re(params.issue_reporter_pattern, content)
 
+        if not (f' Edit Issue:  [{issue_key}]' in content):
+            logger.error(f'{params.err_message_issue_not_found} - {issue_id}, {issue_key}')
```

Comment:
need to add content like:
logger.error(f'User {user[0]} authentication failed: {content}')